content/guides/debugging/_index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Incident Debugging Workflow"
+date: 2018-11-26T22:22:31-07:00
+draft: false
+aliases: [/integrations/workflow]
+weight: 1
+---
+
+##### Using OpenCensus to monitor application health and debug latency issues
```

Comment:
Did you mean `Introduction`?

Reply:
This is meant to be a subtitle for the page. Right now it seems like the menu title has to be the page title but in this case the menu title is too short to describe fully what the page is about.

Reply:
It looks better after I added the ToC that you suggested below and added an Introduction header. Please see new commit.